learn/on-premise/prerequisites.md

```diff
@@ -18,7 +18,7 @@ WaveMaker Enterprise can be installed on any machine with the below requirements
 ### WME Platform Instance
 
 <table><tbody><tr><td><strong>Memory</strong></td><td><ul><li>Minimum 16GB</li></ul></td></tr><tr><td><strong>CPU</strong></td><td><ul><li>4-cores, single CPU system</li><li>Intel Virtualization Technology (VT-x) enabled</li><li>Hyperthreading (HT) disabled</li></ul></td></tr><tr><td><strong>Hard Disk</strong></td><td><ul><li>Minimum&nbsp;250 GB to be allocated</li><li>In case of volumes we recommend 3 disks<ul><li>/&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 50 GB</li><li>/wm-data&nbsp; &nbsp; &nbsp; &nbsp; 150 GB</li><li>/wm-runtime&nbsp; &nbsp;50 GB</li></ul></li></td></tr><tr><td><strong>Host OS</strong></td><td><ul><li>Ubuntu 16.04.6 LTS;  RHEL 7.x/8.x</li><li>Kernel 4.4 or latter</li></ul></td></tr>
-<tr><td><strong>Software</strong></td><td><ul><li>docker 18.06.2-ce</li><li>python 3.5 or higher</li><li>wget</li><li>container-selinux-2.9-4.el7.noarch.rpm(Only for RHEL)</li></ul></td></tr> <tr><td><strong>Network</strong></td><td><ul><li>Static IP with valid DNS</li><li>Ports 80, 443, 8080, 22(for ssh) to developer network range</li><li>Ports to be opened on Platform Instance for Access from StudioWorkspace Instance / AppDeployment Instance<ul><li>Ports : 5000, 8500, 8081, 2200, 9200, 8000-8020</li></ul></li></td></tr></tbody></table>
+<tr><td><strong>Software</strong></td><td><ul><li>docker 18.06.2-ce</li><li>python 3.5 or higher</li><li>wget</li><li>container-selinux-2.9-4.el7.noarch.rpm(Only for RHEL)</li></ul></td></tr> <tr><td><strong>Network</strong></td><td><ul><li>Static IP with valid DNS</li><li>Ports 80, 443, 8080, 22(for ssh) to developer network range</li><li>Ports to be opened on Platform Instance for Access from StudioWorkspace Instance / AppDeployment Instance<ul><li>Ports : 5000, 8500, 22, 8081, 2200, 9200, 8000-8020</li></ul></li></td></tr></tbody></table>
```

Comment:
Only RHEL 7 right ?

Reply:
additional port added 22 is for all , not only for RHEL 7